src/Pie.js

```diff
@@ -3,9 +3,28 @@ import PropTypes from 'prop-types';
 import { Platform } from 'react-native';
 import { Surface, Shape, Path, Group } from '@react-native-community/art';
 
-function createPath(cx, cy, r, startAngle, arcAngle) {
+function createPath(cx, cy, r, startAngle, arcAngle, isBezian, innerRadius) {
   const p = new Path();
-  if (Platform.OS === 'web') {
+  //starting point of our chart
+  if(isBezian && Platform.OS !== 'web'){
+    let ROUNDNESSOUSIDE = 1 - ((r - innerRadius)/ innerRadius) - (arcAngle * .5);
```

Comment:
type. `ROUNDNESSOUTSIDE`

Reply:
if it's mutable, should not use uppercase here